Gemfile.lock

```diff
@@ -566,7 +566,7 @@ DEPENDENCIES
   blacklight_advanced_search!
   blacklight_range_limit
   blacklight_unapi!
-  bootstrap
+  bootstrap (= 4.6.0)
```

Comment:
How was this created ? When I do bundle install it gives me `bootstrap (~> 4.6)`